examples/multiple-choice/README.md

```diff
@@ -18,7 +18,9 @@ limitations under the License.
 
 Based on the script [`run_swag.py`]().
 
-#### Fine-tuning on SWAG
+## PyTorch script: fine-tuning on SWAG
+
+`run_swag` allows you to fine-tune any model from our [hub](https://huggingface.co/models) on the SWAG dataset or your own csv/jsonlines files as long as they are structured the same way. To make it works on another dataset, you will need to tweak the `preprocess_function` inside the script.
```

Comment:
it can't be **any** model, no? any model trained on text-classification perhaps?

Reply:
It's pretty much any model yes, as long as they have a `ForMultipleChoice` head.

Reply:
I found only 18:
```
src/transformers/utils/dummy_pt_objects.py:class AlbertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class AutoModelForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class BertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class CamembertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class ConvBertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class DistilBertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class ElectraForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class FlaubertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class FunnelForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class IBertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class LongformerForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class MobileBertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class MPNetForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class RobertaForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class SqueezeBertForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class XLMForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class XLMRobertaForMultipleChoice:
src/transformers/utils/dummy_pt_objects.py:class XLNetForMultipleChoice:
```

may be it could just say that?

multiple-choice is missing from the tasks on the left side of the hub UI.

Reply:
Will adapt with this, and also will make a nice table showing which model have which head for our documentation one of these days. Once it's there we can link to it.

Reply:
great and for qa it's:

```
grep -Ir ForQuestionAnswering src | grep -v TF | grep dummy
src/transformers/utils/dummy_pt_objects.py:class AlbertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class AutoModelForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class BartForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class BertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class CamembertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class ConvBertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class DebertaForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class DebertaV2ForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class DistilBertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class ElectraForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class FlaubertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class FlaubertForQuestionAnsweringSimple:
src/transformers/utils/dummy_pt_objects.py:class FunnelForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class IBertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class LEDForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class LongformerForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class LxmertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class MBartForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class MobileBertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class MPNetForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class ReformerForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class RobertaForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class SqueezeBertForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class TapasForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class XLMForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class XLMForQuestionAnsweringSimple:
src/transformers/utils/dummy_pt_objects.py:class XLMRobertaForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class XLNetForQuestionAnswering:
src/transformers/utils/dummy_pt_objects.py:class XLNetForQuestionAnsweringSimple:
```